DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -21,6 +21,7 @@ public interface ICourseDataService
         IEnumerable<DelegateCourseInfo> GetDelegateCoursesInfo(int delegateId);
         (int totalAttempts, int attemptsPassed) GetDelegateCourseAttemptStats(int delegateId, int customisationId);
         CourseDetails? GetCourseDetails(int customisationId, int centreId, int categoryId);
+        string GetCourseName(int customisationId);

```

Comment:
Super minor, but the interface return type doesn't match the actual implementation.
![image](https://user-images.githubusercontent.com/40016796/131698656-b5fb9459-300c-4921-b5f7-bb7de26e815b.png)